apps/agent/package.json

```diff
@@ -2,24 +2,20 @@
   "name": "@aragon/apps-agent",
   "version": "2.1.0",
   "scripts": {
-    "compile": "truffle compile",
-    "build": "cd app && npm run build",
-    "apm:prepublish": "npm run compile",
-    "apm:publish:major": "aragon apm publish major --files public/ --prepublish-script apm:prepublish",
-    "apm:publish:minor": "aragon apm publish minor --files public/ --prepublish-script apm:prepublish",
-    "apm:publish:patch": "aragon apm publish patch --files public/ --prepublish-script apm:prepublish",
+    "console": "buidler console",
+    "compile": "buidler compile --force",
+    "devchain": "buidler node",
```

Comment:
We probably don't need this :)

Reply:
Maybe a nice to have. The gas-reporter don't manage the node automatically if we would like to use buidlerevm as explained here: https://github.com/cgewecke/buidler-gas-reporter/tree/master#using-buidlerevm-warning